ext/standard/php_uuid.h

```diff
@@ -0,0 +1,598 @@
+/*
+  +----------------------------------------------------------------------+
+  | PHP Version 7                                                        |
+  +----------------------------------------------------------------------+
+  | Copyright (c) 1997-2017 The PHP Group                                |
+  +----------------------------------------------------------------------+
+  | This source file is subject to version 3.01 of the PHP license,      |
+  | that is bundled with this package in the file LICENSE, and is        |
+  | available through the world-wide-web at the following url:           |
+  | http://www.php.net/license/3_01.txt                                  |
+  | If you did not receive a copy of the PHP license and are unable to   |
+  | obtain it through the world-wide-web, please send a note to          |
+  | license@php.net so we can mail you a copy immediately.               |
+  +----------------------------------------------------------------------+
+  | Author: Richard Fussenegger <php@fleshgrinder.com>                   |
+  +----------------------------------------------------------------------+
+*/
+
+/* $Id$ */
+
+/**
```

Comment:
Like @nikic said in the other PR, we do not use Doxygen and it needs a separate discussion first before we can merge it